app/src/main/java/com/android/wardrobeManager/ui/images/filters/BirdsEye.java

```diff
@@ -0,0 +1,33 @@
+package com.android.wardrobeManager.ui.images.filters;
+
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
+import android.graphics.PorterDuff;
+import android.graphics.PorterDuffXfermode;
+
+import com.android.wardrobeManager.ui.images.DesignFilterManager;
+
+public class BirdsEye implements DesignFilterManager.DesignFilter {
+    int checkerSize = 4;
```

Comment:
This should be passed in through the constructor and should probably have a different name than checker size. Unless you fix my comments in checker first and want to just use that implementation by passing a different value for this. In that case, this class won't be necessary.